src/Traits/ConfigurationTokensTrait.php

```diff
@@ -39,7 +39,11 @@ protected function extractProcessedTokens($text)
         $config = $this->getConfig();
 
         return array_map(function ($key) use ($config) {
-            return $config->get($key);
+            $value = $config->get($key);
+            if (is_array($value)) {
+                return implode(',', $value);
```

Comment:
What do we do if we have nested arrays ?

Reply:
reworked. Thanks!